src/components/pages/ProfilePage.js

```diff
@@ -1,12 +1,34 @@
-import { React } from "react";
+import { React, useContext, useEffect, useState } from "react";
 import { useAuthenticator } from "@aws-amplify/ui-react";
 import { FaArrowRight } from "react-icons/fa";
 import Navbar from "../reusable/NavBar";
 import Sidebar from "../reusable/SideBar";
+import AlgoquantApiContext from "../../ApiContext";
+import LoadSpinner from "../reusable/LoadSpinner";
+import JwtContext from "../../JwtContext";
 
 const ProfilePage = () => {
   const { user } = useAuthenticator((context) => [context.user]);
   const { signOut } = useAuthenticator((context) => [context.user]);
+  const algoquantApi = useContext(AlgoquantApiContext);
+  const jwtContext = useContext(JwtContext);
+  const [balance, setBalance] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+  const [alpacaConnection, setAlpacaConnection] = useState(false);
+
+  useEffect(() => {
+    if (jwtContext) {
+      algoquantApi.getUser(jwtContext).then((resp) => {
```

Comment:
This needs to also have .catch() to handle any errors.

Reply:
should errors get printed into a modal?